### ISSUE TITLE
Keyboard focus is on editor while right click menu is active.

### ISSUE THREAD

**Issue by TinyMCE-User** [tinymce/tinymce#1885]
Description of problem:

Pressing right click in tinymce editor, keyboard focus is still on editor. For e.g you move cursor using arrow keys within editor while right click menu is active.

Steps to reproduce:
1.Right click anywhere in tinymce editor.
2.Press keyboard arrow keys (Up/Down) or Type something.
3.Observe that cursor will traverse in editor or text is written.

Expected result:

After pressing right click, keyboard focus should be on right click menu. For e.g on pressing up/down arrow keys it should traverse right click menus.

Actual result:

Able to traverse cursor using keyboard while right click menu is active.

See attached (.gif)demonstration for more details.

> _**Legacy information imported from TinyMCE bug tracker:**_
> 
> _[#T7155](http://archive.tinymce.com/develop/bugtracker_view.php?id=7155) posted by [sagarkhaire](http://archive.tinymce.com/profile.php?name=sagarkhaire)_
> 
> _**Tags:** [firefox msie safari chrome opera rightclick focus keyboard]_
> _**Status:** Open_
> _**Resolution:** None_
> _**Attached URL:** none_
